docs/templating/twig_function_reference/catalog_twig_functions.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Catalog Twig functions enable getting and rendering certain catalog information and listing filter templates.
+---
+
+# Catalog Twig functions
+
+With the catalog Twig functions you can get catalog location, render catalog status and list filter templates.
+
+### `ibexa_get_filter_preview_templates`
+
+The `ibexa_get_filter_preview_templates()` function gets a list of filter preview Twig templates 
+(configured in `ibexa.repositories.product_catalog.filter_preview_templates`). 
+Then it sorts the list by priority and renders the filter templates.
+
+``` html+twig
+        {% for template in ibexa_get_filter_preview_templates() %}
+            {% if block(block_name, template) is defined %}
+                {{ block(block_name, template) }}
+
+                {% set block_found = true %}
+            {% endif %}
+        {% endfor %}
+```
```

Comment:
@mikadamczyk Please correct me if I'm wrong but this function is consider as internal and doesn't have real use case in customer projects

Reply:
I agree. For me, the only use case is when someone may want to create his own UI for the admin panel